fleet/fleet-with-grpc.md

```diff
@@ -0,0 +1,67 @@
+# Fleet with gRPC
+
+To build fleet with gRPC enabled, first of all, you need to install several
+protobuf compilers.
+
+## Install protobuf packages
+
+Install a native protobuf compiler v3.0.
```

Comment:
@hectorj2f okay, then I'd change it to "Install a native protobuf compiler. Any version should work, but v3.0 is recommended."
I'll also add a link to protobuf documentation.